src/pages/MemoryGame.js

```diff
@@ -54,12 +55,13 @@ const cards = [
 export default Blits.Component('Announcer', {
   components: {
     MemoryCard,
+    Toggle
   },
   template: `
     <Element>
       <Text font="raleway" size="80" color="white" content="Let's play memory!" x="580" y="60" />
       <Text font="kalam" size="40" color="#b91c1c" content="the Accessible edition :)" x="1200" y="170" rotation="-6" />
-    
+      <Toggle ref="toggle" x="100" y="200" :toggled="$toggle" />
```

Comment:
Let's add a small text below to indicate what this toggle is for? Also maybe we should align it a bit to the right to match up with the start of the first card?